src/realm/exec/CMakeLists.txt

```diff
@@ -31,15 +29,24 @@ set_target_properties(RealmBrowser PROPERTIES
 )
 target_link_libraries(RealmBrowser Storage)
 
-add_executable(Realm2JSON EXCLUDE_FROM_ALL realm2json.cpp )
+add_executable(Realm2JSON realm2json.cpp )
 set_target_properties(Realm2JSON PROPERTIES
     OUTPUT_NAME "realm2json-6"
     DEBUG_POSTFIX ${CMAKE_DEBUG_POSTFIX}
 )
 target_link_libraries(Realm2JSON Storage)
+if (NOT WINDOWS_STORE)
+    list(APPEND ExecTargetsToInstall Realm2JSON)
+else()
+    set_target_properties(Realm2JSON PROPERTIES EXCLUDE_FROM_ALL True)
+endif()
```

Comment:
I don't understand this section - what's happening here?

Reply:
I don't entirely understand why, but Realm2JSON had trouble compiling on UWP (see https://ci.realm.io/blue/organizations/jenkins/realm%2Frealm-core/detail/PR-4212/3/pipeline). So, this says if you aren't on UWP, add this to the list of targets to install, otherwise exclude it from building by default.

Reply:
I'll exclude the whole `exec` folder from mobile and UWP builds - we shouldn't have even attempted to build it in the first place.

Reply:
Thanks for catching this.

Reply:
Fixed this so that exec is excluded on UWP.